org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/DefaultWebAppInit.java

```diff
@@ -45,7 +46,9 @@ public void init(WebApp wapp) throws Exception {
 		this.webApp = wapp;
 		// save app name get from zk.xml to restore when delete app name in system config value
 		AdempiereWebUI.APP_NAME = this.webApp.getAppName();
-		String appNameConfig = MSysConfig.getValue(MSysConfig.ZK_BROWSER_TITLE);
+		
+		String fromFile = Env.getContext(Env.getCtx(), Env.PREFIX_SYSTEM_VARIABLE + MSysConfig.ZK_BROWSER_TITLE);
+		String appNameConfig = MSysConfig.getValue(MSysConfig.ZK_BROWSER_TITLE, fromFile);
```

Comment:
isn't the value from env or system properties should take precedence over the value from ad_sysconfig ?

Reply:
That was my first idea, but if I'm not mistaken, Carlos think the opposite.
At the end, as those values are not changed on-the-fly, if the need to have different values, the SysConfig key can be deactivated.
But maybe, you're right, the SysConfig can be considered as the fallback, if not present in the prop file.
I don't know what is the best approach here :-/

Reply:
I don't have a preference on that one.  The suggestion from Heng Sin is maybe more what would be expected.

Reply:
done (and applied also to browser icon)

Reply:
hi @nmicoud , I don't see any changes

Reply:
hi @hengsin, i didn't push the changes :(
is ok now